### PR TITLE
fix: changes removeorder output to a more meaningful message (#1526)

### LINKED ISSUE #1526: feat: Change output of removeorder command/grpc call
### Reason 

UX improvements

### Task 1

Change current output of removeorder to more useful. 
As user I want to know status of the operation, but instead of that I got info about amount in hold (it is additional information, not main)
It seems removed qty should be returned in response + remained qty if it was partial removal + qty in hold if it exists:
{
removedQty:5;# qty that was successfully removed
remainingQty:7;# remaining qty after partial removing
quantityOnHold:5; # qty in swap, but it will be removed if swap fails
}

### Task 2

Add Text output to xucli for removeorder command. Now it returns only json.

Draft:

- order "id" with qty "qty" fully removed - if remained = 0, and quantityOnHold = 0;
- order "id" with qty "qty" partially removed, remained qty = "qty" - if remained != 0, and quantityOnHold = 0;
- order "id" with qty "qty" partially removed, qty on hold because of swap process = "qty" - if remained = 0, and quantityOnHold != 0;

### PR DESCRIPTION
**Pull request by rsercano**
resolves #1526

### CODE REVIEW

**Review by erkarl — DISMISSED** [review]

**Review by raladev — DISMISSED** [review]

File: lib/orderbook/OrderBook.ts [hunk 1]
```diff
@@ -927,7 +934,13 @@ class OrderBook extends EventEmitter {
       this.swaps.on('swap.paid', paidHandler);
     }
 
-    return remainingQuantityToRemove;
+    const remainingFromOrder = orderQuantity - removedQuantity;
+    return {
+      removedQuantity,
+      onHoldQuantity,
+      pairId: order.pairId,
+      remainingQuantity: remainingFromOrder > onHoldQuantity ? remainingFromOrder : 0,
```
Comment (sangaman) [inline lib/orderbook/OrderBook.ts:942]: This seems off to me. If we're trying to indicate the quantity that will remain on the order after any holds or complete, I figure it should be:

```typescript
const remainingQuantity = orderQuantity - removedQuantity - remainingQuantityToRemove;
```

Or perhaps simpler we just use the current quantity on the order (after we've removed whatever we were able to before holds) in which case I don't think we need the `orderQuantity` variable at all.

```typescript
const remainingQuantity = order.quantity - remainingQuantityToRemove;
```

**Review by sangaman — CHANGES_REQUESTED** [review]
See below about how we're calculating remaining quantity, otherwise this looks good to me.

**Review by raladev — APPROVED** [review]

**Review by erkarl — APPROVED** [review]

**Comment by erkarl** [comment]
@sangaman can we get this in?

**Review by sangaman — APPROVED** [review]
Looks good with my last feedback addressed, thanks.

**Pull request merged**